composer.json

```diff
@@ -41,7 +41,8 @@
       "Tests\\ProductFeed\\GoogleBundle\\": "packages/product-feed-google/tests/",
       "Tests\\ProductFeed\\HeurekaBundle\\": "packages/product-feed-heureka/tests/",
       "Tests\\ProductFeed\\HeurekaDeliveryBundle\\": "packages/product-feed-heureka-delivery/tests/",
-      "Shopsys\\MicroserviceProductSearchExport\\": "microservices/product-search-export/src/"
+      "Shopsys\\MicroserviceProductSearchExport\\": "microservices/product-search-export/src/",
+      "Tests\\CodingStandards\\": "packages/coding-standards/tests/"
```

Comment:
This should have been between `"Tests\\"` and `"Tests\\FrameworkBundle\\"` as it is almost-alphabetically sorted.